src/common/personalizedDigest.ts

```diff
@@ -129,10 +131,22 @@ const getEmailVariation = async ({
   const dayName = dayEntry ? dayEntry[0] : undefined;
   const userName = user.name?.trim().split(' ')[0] || user.username;
   const userStreak = await user.streak;
+
+  const posts: Record<string, unknown>[] = getPostsTemplateData({
+    posts: postsData,
+    feature,
+  });
+  if (posts.length >= 2 && !isPlusMember(user.subscriptionFlags?.cycle)) {
```

Comment:
Maybe expose this `2` in digest config so we can experiment?

Reply:
omg! how come i haven't thought about it :thinkaboutit: